compiler/vm/src/tracer/evaluated_values.rs

```diff
@@ -25,7 +25,7 @@ impl EvaluatedValuesTracer {
 impl Tracer for EvaluatedValuesTracer {
     fn value_evaluated(&mut self, heap: &mut Heap, expression: HirId, value: InlineObject) {
         let id = expression.get();
-        if id.module != self.module {
+        if id.module.clone() != self.module {
```

Comment:
The `module.clone()` doesn't seem to be necessary

```suggestion
        if id.module != self.module {
```